.github/workflows/test.yml

```diff
@@ -127,7 +127,7 @@ jobs:
     - name: Run unit tests
       uses: ./
       with:
-        src: tests/src
+        src: tests/src tests/srcB
```

Comment:
I would prefer if you added a new test that tests your behavior, instead of changing every test.

Reply:
ok. this was back to normal in 90d5363